airgun/widgets.py

```diff
@@ -1062,6 +1063,16 @@ def read(self):
         do_not_read_this_widget()
 
 
+class ConfirmationBox(ConfirmationDialog):
+    """PF4 confirmation dialog with two buttons and close 'x' button in the
+    right corner."""
+
+    ROOT = '//div[@id="app-confirm-modal" or @data-ouia-component-type="PF4/ModalContent"]'
```

Comment:
The `or` should not match with the elements that using the same `data-ouia-component-type`. So combine it with `and` instead:
```suggestion
    ROOT = '//div[@id="app-confirm-modal" and @data-ouia-component-type="PF4/ModalContent"]'
```

OR just stick with id.

Reply:
I think it's better to use `or` in case of elements where you are having more robust locator strategies. even in the case, the id changed it will help to locate the element. My guess is that @jameerpathan111 was doing that only   

see e.g. 
![image](https://user-images.githubusercontent.com/3190629/147559537-6944f787-4beb-4a37-bf5c-9d1fac7194dc.png)

Reply:
@jyejare I agree with omkar here.

Reply:
@omkarkhatavkar I was just worried abt `data-ouia-component-type` shouldnt pick any other element! But anyways its a non blocking comment !